sangria/sangria_2.13/src/test/scala/com/linecorp/armeria/server/sangria/Data.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.sangria
+
+object Episode extends Enumeration {
+  val NEWHOPE, EMPIRE, JEDI = Value
+}
+
+trait Character {
+  def id: String
+  def name: Option[String]
+  def friends: List[String]
+  def appearsIn: List[Episode.Value]
+}
+
+case class Human(
+    id: String,
+    name: Option[String],
+    friends: List[String],
+    appearsIn: List[Episode.Value],
+    homePlanet: Option[String])
+    extends Character
+
+case class Droid(
```

Comment:
Is this Scala things? 😄

Reply:
Copied the code from https://sangria-graphql.github.io/learn/#schema-definition 😆